azure-pipelines.yml

```diff
@@ -64,7 +64,7 @@ jobs:
   - job: dist_darwin
     displayName: "Dist Darwin binary"
     pool:
-      vmImage: macOS-10.13
+      vmImage: macOS-10.14
```

Comment:
See comment in my other PR RE: this build fix: https://github.com/jamiebuilds/codeowners-enforcer/pull/9/files#r703770341